src/index.js

```diff
@@ -2,19 +2,13 @@ import '@fortawesome/fontawesome-free/js/all.js';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import './style.css';
-import { createElement } from './modules/utils.js';
 import Todo from './modules/todo.js';
+import { createElement } from './modules/utils.js';
 
 function init() {
-  const tasks = [
-    { description: 'Wash the car', completed: false, index: 0 },
-    { description: 'Goto to grocery store', completed: true, index: 1 },
-    { description: 'Fix the tv', completed: false, index: 3 },
-  ];
-
   const form = createElement('form', {
     innerHTML: `<input
-      type="text" name="todo" id="todo"
+        type="text" required name="todo" id="todo"
```

Comment:
- [ ] Great job so far 👏🏻 However the property index should be set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). Actually, your first book has index equal to 0; instead of 1. Kindly address this 💯 👍🏻 
- Make sure that deleting a task update all remaining items' indexes, so they represent the current list order and are unique. 👍🏻

Reply:
> > @BenMukebo Thanks a lot for your feedback, but I didn't understand what you mean, Those codes are deleted codes.
> > can you please explain a bit more?

This is the point I create a new task, and as you see the new task index is set to tasks.length
https://github.com/BahirHakimy/ToDo-List/blob/daa7c893f7bba1a73cc0639dbaf521c608ee70c9/src/modules/todo.js#L20-L28

Reply:
Hello @BahirHakimy,I was sleeping man 🥲, Let's me explain you well okay 👍🏻

Reply:
> Thank you @BenMukebo for clarification, I will correct this issue.